go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1766,6 +1767,27 @@ func FnLastDay(yield Query) {
 	}
 }
 
+func FnToDays(yield Query) {
+	for _, d := range inputConversions {
+		yield(fmt.Sprintf("TO_DAYS(%s)", d), nil)
+	}
+
+	dates := []string{
+		`DATE'0000-00-00'`,
+		`0`,
+		`'0000-00-00'`,
+		`DATE'2023-09-03 00:00:00'`,
+		`DATE'2023-09-03 07:00:00'`,
+		`DATE'0000-00-00 00:00:00'`,
+		`950501`,
+		`'2007-10-07'`,
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_to-days has these cases:

```
TO_DAYS(950501);
TO_DAYS('2007-10-07');
TO_DAYS('2008-10-07'),
TO_DAYS('08-10-07');
TO_DAYS('0000-00-00');
TO_DAYS('0000-01-01');
```

Can you add the ones from there missing to make sure they all pass?

Reply:
done.